src/website/views/components/popover/_examples.pug

```diff
@@ -884,6 +884,41 @@ h3 Destructive action confirmation
                 i.chi-icon.icon-chevron-down(aria-hidden="true")
               .chi-accordion__content
                 p.chi-accordion__text Filter content goes here
+            .chi-accordion__item
+              button.chi-accordion__trigger
+                .chi-accordion__title
+                  | Accordion 5
+                i.chi-icon.icon-chevron-down(aria-hidden="true")
+              .chi-accordion__content
+                p.chi-accordion__text Filter content goes here
+            .chi-accordion__item
+              button.chi-accordion__trigger
+                .chi-accordion__title
+                  | Accordion 6
+                i.chi-icon.icon-chevron-down(aria-hidden="true")
+              .chi-accordion__content
+                p.chi-accordion__text Filter content goes here
+            .chi-accordion__item
+              button.chi-accordion__trigger
+                .chi-accordion__title
+                  | Accordion 7
+                i.chi-icon.icon-chevron-down(aria-hidden="true")
+              .chi-accordion__content
+                p.chi-accordion__text Filter content goes here
+            .chi-accordion__item
+              button.chi-accordion__trigger
+                .chi-accordion__title
+                  | Accordion 8
+                i.chi-icon.icon-chevron-down(aria-hidden="true")
+              .chi-accordion__content
+                p.chi-accordion__text Filter content goes here
+            .chi-accordion__item
+              button.chi-accordion__trigger
+                .chi-accordion__title
+                  | Accordion 9
```

Comment:
Let's add a couple more to get more prominent scroll.

Reply:
OK! Added